services/web/server/src/simcore_service_webserver/director_v2.py

```diff
@@ -72,6 +72,19 @@ async def _request_director_v2(
         ) from err
 
 
+async def is_healthy(app: web.Application) -> bool:
+    try:
+        session = get_client_session(app)
```

Comment:
make sure this has a very low timeout

Reply:
The design of the webserver imposes a single aiohttp.ClientSession for the entire application and timeout settings are also defined at an app level. Therefore, those are the timeouts defined here.

- I agree that the health-checks on backend services should not be sequential and added concurrency 
- I reduced the calls by providing ``url`` fields to other entrypoint to get more detail  (HATEOS approach)
- Added a shorted timeout for "health-check" calls